doc/md/versioned-migrations.md

```diff
@@ -50,101 +61,52 @@ func main() {
 }
 ```
```

Comment:
A suggestion to embrace static typing and use the `gen.FeatureVersionedMigration`.